kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosButton.vue

```diff
@@ -38,7 +38,7 @@
         </div>
       </template>
       <span>
-        {{ buttonDisabled && $t('exoplatform.kudos.info.onlyOtherCanSendYouKudos') || $t('exoplatform.kudos.title.sendAKudos') }}
+        {{ buttonDisabled === 'same' && $t('exoplatform.kudos.info.onlyOtherCanSendYouKudos') || buttonDisabled === 'inactive' && $t('exoplatform.kudos.title.sendAKudos') }}
```

Comment:
```suggestion
        {{ buttonDisabled && ((this.comment.identity.id === this.userIdentityId &&   $t('exoplatform.kudos.info.onlyOtherCanSendYouKudos')) || ((inactiveCommentOwner()  || inactiveActivityOwner()) && $t('exoplatform.kudos.title.sendAKudos')) }}
```

Reply:
IMO the best solution for this case is to use the common string return for activity and comment 'inactive'/'same' to ensure proper display for the label
else if I do it so, and if the first check is true other checks will be ignored, exp "((inactiveCommentOwner  || inactiveActivityOwner) && $t('exoplatform.kudos.title.sendAKudos'))"

Reply:
@hthabti Could you update the code with your proposed change ?

Reply:
@ahamdi proposition added